src/include/processor/operator/copy/copy_npy.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "processor/operator/copy/copy.h"
+#include "storage/store/nodes_store.h"
+
+namespace kuzu {
+namespace processor {
+
+class CopyNPY : public Copy {
```

Comment:
This is confused. Can we make NPY as a reader and reuse CopyNode/CopyRel operator? Otherwise, we would need to have CopyNPYNode and CopyNPYRel, which is incorrect to me.